docs/ops/movement/Transpose_1.md

```diff
@@ -6,30 +6,28 @@
 
 **Short description**: *Transpose* operation reorders the input tensor dimensions.
 
-**Attributes**:
+**Detailed description**: *Transpose* operation reorders the input tensor dimensions. Source indexes and destination indexes are bound by the formula: 
 
-No attributes available.
+    output[i(order[0]), i(order[1]), ..., i(order[N-1])] = input[i(0), i(1), ..., i(N-1)], 
+    where i(j) in range 0..(input.shape[j]-1)
```

Comment:
Ranges are better represented by intervals
```suggestion
    where i(j) is within the range `[0, (input.shape[j]-1)]`
```

Reply:
It was a part of the math formula, but done:)